database/errors.go

```diff
@@ -117,7 +118,7 @@ func NewExecutionError(action SQLQueryType, stmt string, err error, logger log.L
 
 // Error prints a ExecutionError
 func (e *SQLExecutionError) Error() string {
-	return fmt.Sprintf("%s (%s)", e.message, e.ReferenceID)
+	return fmt.Sprintf("%s: %s (%s)", e.message, e.ErrMsg, e.ReferenceID)
```

Comment:
let me look at these, but generally they give away too much information. We do not want people using the API to be able to find out anything about our system (OS, DB, Language etc). This ticket would be more for returning the name of the field there was a problem with via expanding the REST Error and converting correctly there.